core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -98,13 +97,12 @@ private[spark] class Pool(
   }
 
   override def getSortedTaskSetQueue: ArrayBuffer[TaskSetManager] = {
-    var sortedTaskSetQueue = new ArrayBuffer[TaskSetManager]
-    val sortedSchedulableQueue =
-      schedulableQueue.asScala.toSeq.sortWith(taskSetSchedulingAlgorithm.comparator)
-    for (schedulable <- sortedSchedulableQueue) {
-      sortedTaskSetQueue ++= schedulable.getSortedTaskSetQueue
-    }
-    sortedTaskSetQueue
+    schedulableQueue
```

Comment:
This isn't a minor typo change. I'm not clear this will work? before it was producing a collection from the individual schedulable's queues but that has disappeared. Is this even of the right type?